code/modules/surgery/organs_internal.dm

```diff
@@ -122,14 +122,26 @@
 			var/obj/item/organ/I = target.internal_organs_by_name[organ]
 			if(I && !(I.status & ORGAN_CUT_AWAY) && I.parent_organ == target_zone)
 				attached_organs |= organ
+		if(!attached_organs.len)
+			return 0
+
+
+
+		return ..()
 
+
+	prepare_step(mob/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
```

Comment:
These three organ manipulating procs got a prepare_step because they ask the user which organ to work on. 
they were asking in can_use and this was causing it to ask twice (with only the second time counting) because can_use is called twice before starting.